Detectors/GLOQC/src/MatchITSTPCQC.cxx

```diff
@@ -516,10 +547,29 @@ void MatchITSTPCQC::run(o2::framework::ProcessingContext& ctx)
   mTPCTracks = mRecoCont.getTPCTracks();
   mITSTracks = mRecoCont.getITSTracks();
   mITSTPCTracks = mRecoCont.getTPCITSTracks();
+  mPVs = mRecoCont.getPrimaryVertices();
 
   LOG(info) << "****** Number of found ITSTPC tracks = " << mITSTPCTracks.size();
   LOG(info) << "****** Number of found TPC    tracks = " << mTPCTracks.size();
   LOG(info) << "****** Number of found ITS    tracks = " << mITSTracks.size();
+  LOG(info) << "****** Number of PVs                 = " << mPVs.size();
+
+  const auto& pvm = mRecoCont.getPrimaryVertexMatchedTrackRefs();
+  for (int i{0}; i < mPVs.size(); ++i) {
+    const auto& pv = mPVs[i];
+    const auto& m = pvm[i];
+    // count the number of ITS tracks that were associated to this PV
+    // then one can plot the number of tracks contributing to a PV against the available pool
+    // (+ ITS tracks only in OB which by constructing cannot contribute).
+    Double_t c{0.};
+    for (int src = GIndex::NSources; src--;) {
+      if (!GIndex::includesDet(DetID::ITS, src)) {
+        continue;
+      }
+      c += m.getEntriesOfSource(src);
+    }
+    ((mIsHI) ? mPVNContVsITSTracksPbPb : mPVNContVsITSTracks)->Fill(pv.getNContributors(), c);
```

Comment:
Hi @shahor02, does the accounting look correct to you?

Reply:
ping @shahor02 :)

Reply:
Sorry, missed the question... 
If the aim is to show the evolution of the pool size vs ncontrib, it is fine, but they will be strongly correlated.
The limits of histos look bit strange: the number of ITS time-associated tracks should be only a few 10% higher than the contributors, not an order of magnitude.

Reply:
Thanks, actually I should have not pinged you, sorry about that. Since I will not be available for the next three weeks, I will close this and afterward continue working on this :)